blinkit-app/src/homePage/molecules/foodbar/Foodbar.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import InsertList from "../../atoms/insertList/InsertList";
+import InsertSelect from "../../atoms/insertSelect/InsertSelect";
+import { foodbarList,moreValue } from "../../../DATA/FOODBAR_DATA";
+import "./foodbar.css";
+class Foodbar extends Component {
+  render() {
+    return (
+      <section className="foodbar">
+        <ul>
```

Comment:
This should be a responsibility of your List component, if needed take user input to decide whether the list should be ordered or unordered